### ISSUE TITLE
(Qol/Suggestion) Waste decay, slower automatic rad cleanup, variants of Baleonite

### ISSUE THREAD

**Issue by xenoulexi** [Alcatergit/Hbm-s-Nuclear-Tech-GIT#328]
Firstly, I think it would be cool and useful if blocks of nuclear waste decayed after being placed down for a few hours, turning into something more useful that could be centrifuged for fission products, it would give the players a reason to build nuclear waste disposal sites instead of simply deleting the nuclear waste.
Secondly, as much as I like the idea of the automatic radiation cleanup setting, I think it happens way too quickly, contaminated blocks should take multiple hours to become safe again, and instead of turning back into the blocks they were before being contaminated I think they should still be permanently altered, just no longer radioactive. Contaminated dirt could turn into dead dirt, charred wood could crumble into ashes, fallout could turn into a black sand, and sellafite/baleonite could slowly go from the "hotter" variants to the "colder" variants, eventually all of it just transmuting into slaked sellafite after enough time. Sellaline could harden into blocks of sellafite after a few minutes, and over very long periods of time, like dozens of hours, corium and coreblestone could also decay into something much safer to handle.
I'd like it if Baleonite got variants like Sellafite has gotten, and for balefire explosions to spawn a lot more balefire flames, perhaps balefire bombs could also have their own version of fallout.

**Comment by Shibva** [comment]
erm; I hate to break it to you but if you have noticed, certain things run on a semi-realistic timescale of Minecraft itself (24000 ticks, a full day, is about 20 IRL minutes. RTG pellets decay realistically in this timeframe on par to how it would decay in real life( so using what you said here, it would take a long time for most things in game to decay by that logic unless the thing in question has a very short half life and the shortest ive seen is the balefire-XXX pelet so take that as you will

on another note their is a drum (not the cooling one) that you can put stuff into it to decay into other things (Fun fact: irradiated items can also be put in here to have their contamination irradiate away); Gold 198 is an example of this as it will decay into mercury

**Comment by xenoulexi** [comment]
Yeah, I think it would be neat if there was a config option to multiply decay times. So you can have stuff decay x3 or x6 times the normal rate.

**Comment by Shibva** [comment]
all of them work on ticks so... I think you can speed it up if you give them extra ticks via tick accelerators like the TIme in a Bottle for example
